charts/umbrella/values.yaml

```diff
@@ -772,20 +795,20 @@ dataconsumerTwo:
   seedTestdata: false
   nameOverride: dataconsumer-2
   secrets:
-    edc-miw-keycloak-secret: changeme  # TODO switch to existing user
+    edc-miw-keycloak-secret: WUXpQx1aIclA7enqtk4o2uvLDLMreUMI
```

Comment:
could you please take tPwy4exxH1sXBRQouobSA2nNVaaPuwCs , same as in adopter-dataexchange values file